### ISSUE TITLE
Docs: upgrade the documentation for contribution guidelines

### ISSUE THREAD

**Issue by Sanchitbajaj02** [Sanchitbajaj02/palettegram#19]
### What's wrong in the documentation?

The documentation for contribution guidelines lack some features and I want that to be enhanced

### Add screenshots

N/A

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I want to work on this issue

**Comment by ashishk1331** [comment]
Is this free tot take?

**Comment by Sanchitbajaj02** [comment]
@ashishk1331 yeah sure. I updated most of the things but if you feel something is missing, you can add it. I am assigning it to you.
